specification/communication/data-plane/Chat/stable/2021-09-07/communicationserviceschat.json

```diff
@@ -1688,6 +1700,38 @@
           "example": "Bob Admin"
         }
       }
+    },
+    "DeletionStrategy": {
+      "type": "object",
+      "discriminator": "strategy",
+      "properties": {
+        "strategy": {
```

Comment:
Can you make this an enum? See [EventGrid](https://github.com/Azure/azure-rest-api-specs/blob/56435032b69cc7ebb5c1052b66c7865d0c48f33c/specification/eventgrid/resource-manager/Microsoft.EventGrid/stable/2019-01-01/EventGrid.json#L1783-L1796) for example.

Reply:
Can you also add a "description" property here?